front/src/styles/scss/applications/operationalStudies/_study.scss

```diff
@@ -234,59 +270,95 @@
+    &:hover, &:focus, &:active {
+      border-color: var(--coolgray11);
+      .scenarios-list-card-name {
+        color: var(--coolgray3);
+        background-color: var(--coolgray11);
+      }
+      button {
+        opacity: 1 !important;
+      }
     }
     button {
-      opacity: 1 !important;
+      transition: opacity .4s;
+      opacity: 0;
+      margin-left: auto;
     }
-  }
-  button {
-    transition: opacity .4s;
-    opacity: 0;
-    margin-left: auto;
-  }
-  .scenarios-list-card-name {
-    display: flex;
-    align-items: center;
-    color: var(--coolgray11);
-    background-color: var(--coolgray3);
-    padding: .25rem .25rem .25rem .5rem;
-    font-size: 1.1rem;
-    transition: .4s;
-  }
-  .scenarios-list-card-description {
-    padding: .5rem;
-    font-size: .9rem;
-    height: 6rem;
-    overflow: auto;
-  }
-  .scenarios-list-card-footer {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    .scenarios-list-card-infra {
-      padding: .25rem .5rem;
-      font-size: .8rem;
-      img {
-        height: 1rem;
-        margin-right: .5rem;
+    .scenarios-list-card-name {
+      display: flex;
+      align-items: center;
+      color: var(--coolgray11);
+      background-color: var(--coolgray3);
+      padding: .25rem .25rem .25rem .5rem;
+      font-size: 1.1rem;
+      transition: .4s;
+    }
+    .scenarios-list-card-description {
+      padding: .5rem;
+      font-size: .9rem;
+      height: 6rem;
+      overflow: auto;
+    }
+    .scenarios-list-card-tags {    
+      display: flex;
+      align-items: center;
+      flex-wrap: wrap;
+      gap: .25rem;
+      padding: 0 0.5rem;
+      height: 1.5rem;
+      overflow: auto;
+      .scenarios-list-card-tags-tag {   
+        font-size: .8rem;
+        font-weight: 500;
+        color: var(--secondary);
+        background-color: var(--teal);
+        border-radius: 4px;
+        padding: 0 0.25rem;
+        transition: 0.2s;
+        cursor: pointer;
+        &:hover {
+          opacity: 0.65;
+        }
       }
     }
-    .scenarios-list-card-date {
-      color: var(--coolgray9);
-      font-size: .6rem;
-      text-transform: uppercase;
-      text-align: right;
-      padding: .25rem .5rem;
+    .scenarios-list-card-footer {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      .scenarios-list-card-infra {
+        padding: .25rem .5rem;
+        font-size: .8rem;
+        img {
+          height: 1rem;
+          margin-right: .5rem;
+        }
+      }
+      .scenarios-list-card-trains-count, .scenarios-list-card-date {
+        display: flex;
+        align-items: baseline;
+        height: 1rem;
+        white-space: nowrap;
+        color: var(--coolgray9);
+        font-size: .6rem;
+        text-transform: uppercase;
+        text-align: right;
+        padding: .25rem .5rem;
+        svg {
+          margin-bottom: 0.15rem;
+        }
+      }
     }
   }
 }
```

Comment:
missing new line

Reply:
Corrected all end lines in scss (activated option in vscode)